pkg/diagnose/diagnose.go

```diff
@@ -103,6 +104,25 @@ func (c *Collector) CollectCR(list runtime.Object) {
 	}
 }
 
+// collect output of the "describe pod"
+func (c *Collector) collectPodDescribe(pod *corev1.Pod) {
+	cmd := exec.Command("kubectl", "describe", "pod", "-n", pod.Namespace, pod.Name)
```

Comment:
noobaa has a `--kubeconfig` option. I think it is handled here:

https://github.com/dannyzaken/noobaa-operator/blob/23f9a9b4523e0aaeb0968660e3f57a5a62deacab/pkg/util/util.go#L103-L115


when calling kubectl we should use the same kubeconfig

Reply:
Sounds good, done.